ml-dhs-aws/src/test/java/com/marklogic/client/dhs/test/TestJobRunner.java

```diff
@@ -48,17 +70,19 @@ public void testRun() throws IOException {
       String jobFile = JobRunner.jobFileFor(csvFile);
 
       String jobCollection = null;
+      TestJobLogger testJobLogger = new TestJobLogger();
 
       JobRunner jobRunner = new JobRunner();
-      jobRunner.setLogger(logger);
       jobRunner.setRoles(role);
 
       try (
             InputStream csvStream = openStream(csvFile);
             InputStream jobStream = openStream(jobFile);
             InputStream jobStream2 = openStream(jobFile)
       ) {
-         jobRunner.run(client, csvStream, jobStream);
+         jobRunner.run(client, csvStream, jobStream,testJobLogger);
```

Comment:
Does the story also specify passing in the estimated total bytes for logging in the status message?  Something like:
```
File csvFilehandle = new File(TestJobRunner.class.getClassLoader().getResource(csvFile).toURI());
long totalBytes = csvFilehandle.length();
```
modifying the try-with-resources line to
`InputStream csvStream = new FileInputStream(csvFilehandle);`
Would something along those lines work?